src/main/java/org/gridsuite/modification/dto/LccCreationInfos.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.dto;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.iidm.network.HvdcLine;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.dto.annotation.ModificationErrorTypeName;
+import org.gridsuite.modification.modifications.AbstractModification;
+import org.gridsuite.modification.modifications.LccCreation;
+
+/**
+ * @author Ghazwa Rehili <ghazwa.rehili at rte-france.com>
+ */
+
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(callSuper = true)
+@Schema(description = "LCC creation")
+@JsonTypeName("LCC_CREATION")
+@ModificationErrorTypeName("CREATE_LCC_ERROR")
+public class LccCreationInfos extends EquipmentCreationInfos {
```

Comment:
I am not why there is a Lcc ? I tkink Lcc is juste the name for the converterStations between the station it is a hvdc line

Reply:
it is the same class as VscCreationInfos except for convert station 
there should be a HvdcCreationInfo Class with converterStations as interface that could either lcc or vsc